src/api/app/views/webui/users/tokens/show.html.haml

```diff
@@ -0,0 +1,64 @@
+- @pagetitle = 'Show Token'
+
+.card
+  .card-body
+    %h3.mb-3= @pagetitle
```

Comment:
If the token's name is defined, I suggest displaying here. If not, I suggest displaying _No name_ as we do on the tokens index page. We should change the `@pagetitle` to reflect this in the view and in the browser's window title.